DabraHalleloEndaAbunaSamuel/DHAS018.xml

```diff
@@ -0,0 +1,131 @@
+                                                       <measure unit="leaf"/>                                  
+                                                       <dimensions type="outer" unit="mm">
+                                                           <height cert="low">280</height>
+                                                           <width cert="low">240</width>
+                                                           <depth cert="low">80</depth>
+                                                           
+                                           </dimensions>
+                                       </extent>   
+                            
+                                                     
+                         </supportDesc>
+            
+                         <layoutDesc>
+                             <layout columns="2">                                 
+                             </layout>
+                         </layoutDesc>
+                         
+                     </objectDesc>
+                       <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                               <seg type="script" cert="medium">20th-/21st-century script.</seg>
+                                                        
+                           </handNote>                           
+                       </handDesc>                       
+                       <decoDesc>
+                           <decoNote type="miniature" xml:id="d1">
+                             <desc>A crude miniatury of St Mary in the beginning</desc>
+                           </decoNote>
+                       </decoDesc>
+                       
+                         <bindingDesc>
+                             <binding contemporary="true">
+                                 <decoNote xml:id="b1" type="Boards" cert="medium">Wooden boards.</decoNote>                                
+                                 <decoNote xml:id="b2" type="Cover" cert="medium">Blind-tooled leather cover, 
+                                     <term key="textileInlay">textile inlays</term>.</decoNote>
+                                 <decoNote xml:id="b3" type="bindingMaterial">
+                                     <material key="wood"/>
+                                     <material key="leather"/>
+                                     <material key="textile"/>
+                                 </decoNote>                                     
+                 
+                                 
+                             </binding>
+                         </bindingDesc>
+                   </physDesc>                     
+                  
+                    
+                      <history>
+                         <origin>
+                           <origDate notBefore="1900" notAfter="2020">Recent</origDate>.                              
```

Comment:
As above: "20th/21st century, but not after 2020".

Reply:
ok